polar2grid/readers/mirs.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+# Copyright (C) 2012-2015 Space Science and Engineering Center (SSEC),
+# University of Wisconsin-Madison.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# This file is part of the polar2grid software package. Polar2grid takes
+# satellite observation data, remaps it, and writes it to a file format for
+#     input into another program.
+# Documentation: http://www.ssec.wisc.edu/software/polar2grid/
+#
+# Written by Joleen Feltz    January 2021
```

Comment:
Now that there is more than one person contributing to this project :wink: maybe we should just drop this whole last portion. I find that they become out of date or don't represent the most recent authorship/maintainership of a module.

Reply:
Oh Yes.  I can drop that.  I was wondering about that.